app/models/input_element.rb

```diff
@@ -95,15 +96,15 @@ def parsed_name_for_admin
   end
 
   def has_flash_movie # rubocop:disable Naming/PredicateName
-    description.try :embeds_player?
+    description_embeds_player?
   end
 
   # For loading multiple flowplayers classname is needed instead of id
   # added the andand check and html_safe to clean up the helper
   #
   def sanitized_description
     ie8_sanitize(
-      description.try(:sanitize_embedded_player)
+      description_sanitize_embedded_player
```

Comment:
I wonder if we can drop this entirely? The only `<object>` in the input element descriptions is a YouTube video. I'm pretty certain we don't use Flowplayer anymore.

Reply:
I only count 5 occurences of Flowpayer videos in all of the descriptions (and I believe those videos to be hidden either way), so I think it's safe to drop it. I could also remove those 5 video links from the descriptions, to be totally safe.